examples/collectionless-item.json

```diff
@@ -73,20 +73,14 @@
   },
   "collection": "CS3",
```

Comment:
Should the `"collection"` field be removed as well?

Reply:
Yes, indeed. Thanks!